pkg/drivers/qemu/qemu.go

```diff
@@ -402,41 +403,48 @@ func (d *Driver) Start() error {
 		"-pidfile", d.pidfilePath(),
 	)
 
-	if d.Network == "user" {
+	switch d.Network {
+	case "user":
 		startCmd = append(startCmd,
 			"-nic", fmt.Sprintf("user,model=virtio,hostfwd=tcp::%d-:22,hostfwd=tcp::%d-:2376,hostname=%s", d.SSHPort, d.EnginePort, d.GetMachineName()),
 		)
-	} else if d.Network == "tap" {
+	case "tap":
 		startCmd = append(startCmd,
 			"-nic", fmt.Sprintf("tap,model=virtio,ifname=%s,script=no,downscript=no", d.NetworkInterface),
 		)
-	} else if d.Network == "vde" {
+	case "vde":
 		startCmd = append(startCmd,
 			"-nic", fmt.Sprintf("vde,model=virtio,sock=%s", d.NetworkSocket),
 		)
-	} else if d.Network == "bridge" {
+	case "bridge":
 		startCmd = append(startCmd,
 			"-nic", fmt.Sprintf("bridge,model=virtio,br=%s", d.NetworkBridge),
 		)
-	} else {
-		log.Errorf("Unknown network: %s", d.Network)
+	default:
+		err := fmt.Errorf("unknown network: %s", d.Network)
+		log.Error(err)
+		return err
```

Comment:
I will remove this until we can confirm whether a network is required, or defaults to "user".